apps/studio/prisma/generate.cts

```diff
@@ -0,0 +1,136 @@
+import * as fs from "fs/promises"
+import * as path from "path"
+import ts from "typescript"
+
+const TYPE_FOLDER = "./generated"
+// THIS MUST BE PREFIXED WITH A DOT
```

Comment:
does this comment apply only to `TYPE_FILE` or to the group of `const`? seems like it applies to all

Reply:
idk. i copied from activesg implementation. dexter write one LOL